RELEASE.md

```diff
@@ -0,0 +1,10 @@
+## Procedures
+* make a PR to update:
+  * kube-state-metrics image tag for both `quay.io` and `k8s.gcr.io`
+  * compatibility matrix
+  * change log
```

Comment:
change log -> "changelog entry" ; Also I like to write the changelog to only include user relevant changes. Also I like to keep the same order of changelog entries:

Entries in the CHANGELOG.md are meant to be in this order:

[CHANGE]
[FEATURE]
[ENHANCEMENT]
[BUGFIX]

Last but not least, all lines in the changelog should be full sentences.

Reply:
Done.

Btw, the types of change in [keepchangelog](http://keepachangelog.com/en/1.0.0/) are 
```
Added
Changed
Deprecated
Removed
Fixed
Security
```

In case we have sticked to use `[CHANGE]/[FEATURE]/[ENHANCEMENT]/[BUGFIX]`, i am ok we keep using those types.

Reply:
change, feature, enhancement, bugfix is what we used so far, so let's stick with it.